src/commoncode/urn.py

```diff
@@ -85,9 +85,12 @@
 The product object type syntax is the same as the component syntax.
 """
 
-from urllib import quote_plus
-from urllib import unquote_plus
+#The future package makes the Python 3.x APIs available on Python 2.x
+from future.standard_library import install_aliases
+install_aliases()
```

Comment:
Could use something that would not depend on `future`: I would rather have a simple try/except on imports.
Also check if this `urllib` is in use anywhere else that would need fixing and apply that one related fix everywhere.